<ui version="4.0" >
 <class>phonon</class>
 <widget class="QWidget" name="phonon" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="1" column="0" >
    <widget class="QWidget" native="1" name="wVideo" />
   </item>
   <item row="2" column="0" >
    <widget class="QFrame" name="frControl" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QToolButton" name="bStop" >
      <property name="geometry" >
       <rect>
        <x>115</x>
        <y>7</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text" >
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="bPlay" >
      <property name="geometry" >
       <rect>
        <x>251</x>
        <y>7</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text" >
       <string>...</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QWidget" native="1" name="widget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Audio</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="cbAudio" />
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Subtitles</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QComboBox" name="cbSubtitle" />
      </item>
      <item row="0" column="5" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>125</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbSubtitle</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>phonon</receiver>
   <slot>cbSubtitleChanged(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>345</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAudio</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>phonon</receiver>
   <slot>cbAudioChanged(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>86</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cbSubtitleChanged(int)</slot>
  <slot>cbAudioChanged(int)</slot>
 </slots>
</ui>
